docs/manual.rst

```diff
@@ -499,8 +500,6 @@ Figure 4. On the left, a `Polygon` that is `invalid` because its exterior and
 interior rings touch along a line, and on the right, a `Polygon` that is
 `invalid` because its interior rings touch along a line.
 
-A `Polygon` has non-zero area and non-zero length.
```

Comment:
This was removed on purpose? (I know it is not entirely correct because an empty polygon has a zero area .. but this sentence also served as intro to the next code block showing the area and length attribute)

Reply:
Yes, the issue being resolved was the comment that this sentence is misleading as it implies that an empty Polygon is invalid which is not the case...

Maybe this gives the best of both worlds?
`A `Polygon` has non-zero area and non-zero length if it isn't empty.`